frontend/components/EpisodeTemplate.js

```diff
@@ -68,8 +68,8 @@ const useStyles = makeStyles({
     fontFamily: "Montserrat, sans-serif",
     color: "#313638",
     "@media (prefers-color-scheme:dark)": {
-      color: "#fff", 
-    }
+      color: "#fff",
+    },
```

Comment:
Resolve the following console error 
![image](https://user-images.githubusercontent.com/80564580/188311919-f561ce02-93df-4e3a-8cae-b53ecaaf57cb.png)